packages/marathon/buildinfo.json

```diff
@@ -4,8 +4,8 @@
   ],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://downloads.mesosphere.io/marathon/builds/1.8.227-8615e776b/marathon-1.8.227-8615e776b.tgz",
-    "sha1": "baccf81c2c62ed956b756e5b0291d8767fb1ca2d"
+    "url": "https://s3.amazonaws.com/downloads.mesosphere.io/marathon/builds/1.8.230-dd7806a1f/marathon-1.8.230-dd7806a1f.tgz",
```

Comment:
for future reference, i think its best to use `https://downloads.mesosphere.com/.....` which the cloudfront for the `downloads.mesosphere.io` amazon s3 bucket.

Reply:
ref - https://mesosphere.slack.com/archives/C5BRLULBF/p1543388172177800